### PR TITLE
horoscope feature added

### PR DESCRIPTION
**Pull request by pahosler**
please review

### CODE REVIEW

File: src/commands/fun/horoscope.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+// @Author Paul Hosler
+// required modules for this feature
+const {Command} = require('discord.js-commando');
+// const fetch = require('node-fetch');
+const logger = require('../../utils/logging.js');
+const getHoroscope = require('../../utils/getHoroscope.js');
```
Comment (glynnw) [inline src/commands/fun/horoscope.js]: You seemed to have left this out of the pull request.

File: src/bot.js [hunk 2]
```diff
@@ -1,34 +1,33 @@
...
-    ["humor", "Bot's humor processing unit"],
-    ["scrapers", "Scrape for data from remote sources"],
-    ["fetch", "Fetch data from websites"]
+    ['music', 'Music related commands'],
+    ['code', 'Code related commands'],
+    ['assistants', 'Commands to assist the bot overlord(s)'],
+    ['humor', "Bot's humor processing unit"],
+    ['scrapers', 'Scrape for data from remote sources'],
+    ['fetch', 'Fetch data from websites'],
+    ['fun', 'Games our bot likes to play'],
```
Comment (glynnw) [inline src/bot.js]: It would be better to just stage this line and leave the style changes (single quote from double quotes) to separate commit so as to not hide them in an unrelated commit.

File: src/bot.js [hunk 3]
```diff
@@ -1,56 +1,55 @@
...
-  .registerDefaultGroups()
-  .registerDefaultCommands({
-    eval: false
-  })
-  .registerCommandsIn(path.join(__dirname, 'commands'))
+      ["music", "Music related commands"],
+      ["code", "Code related commands"],
+      ["assistants", "Commands to assist the bot overlord(s)"],
+      ["humor", "Bot"
+        s humor processing unit "], ["scrapers", "Scrape for data from remote sources"],
```
Comment (glynnw) [inline src/bot.js]: You introduced a break here.

**Comment by jodylecompte** [comment]
This looks good to me.
